server/config/index.js

```diff
@@ -15,16 +17,19 @@ if (process.env.GRAPHIQL_QUERY_FILE) {
   }
 }
 
+const port = process.env.HTTP_PORT || '8000'
```

Comment:
Add all in the top without space.

Reply:
Variable declarations at the top are typically used for declaring imports.

Reply:
@darahayes at on top typically is declared all that is global to the program in all languages. For default, it follows the following standard/convention.

```javascript
imports ...
global vars ...
functions/methods ...
```

It is something important to make easier to keep the project maintained, global vars in many places make harder to read and understand the code as could cause mistakes, for example, declare the same value again in another global var. 

Note that this standard is valid for any language. `global variables are often available by declaring a variable at the top level of the program.` See [here](https://en.wikipedia.org/wiki/Global_variable)

Also, it is a good practice to avoid global variables in JS and try to use always local variable instead of it. Following some references.

* https://www.w3schools.com/js/js_best_practices.asp
* https://www.w3.org/wiki/JavaScript_best_practices